doc/scapy/usage.rst

```diff
@@ -403,7 +403,7 @@ In order to quickly review responses simply request a summary of collected packe
 
 The above will display stimulus/response pairs for answered probes. We can display only the information we are interested in by using a simple loop:
 
-    >>> ans.summary( lambda(s,r): r.sprintf("%TCP.sport% \t %TCP.flags%") )
```

Comment:
I'd rather have `lambda s, r:`

Reply:
I tried that first, but seems like the lambda is called with one argument (a tuple) so that would throw a missing argument error.

Reply:
Yeah, so the way to go is to migrate to `lambda s, r:` and use `scapy.compat.lambda_tuple_converter` where it is called. Could you do that?

Reply:
I see, thanks, will give that a try.

Reply:
Actually not sure I follow. When we call [summary](https://github.com/secdev/scapy/blob/7eac62d71c241df8d3f24b8fd89a7c057845cbee/scapy/plist.py#L140) it is just passing some list element to the lambda, but we want to have the lambda with 2 arguments, in which case `lambda_tuple_converter` won't have any effect (seems like it's meant to go the other direction?). Am I missing something?